CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
```

Comment:
Since this is a hot fix, can you up the version number to 1.10.1 everywhere:

- Changelog
- Built.scala
- swagger.yml
- conf.py

I usually just search for the prior version number, in this case 1.10.0.

Reply:
updated in those places